src/test/java/com/deque/axe/ExampleTest.java

```diff
@@ -179,18 +180,23 @@ public void testAccessibilityWithIncludesAndExcludes() {
 	 */
 	@Test
 	public void testAccessibilityWithWebElement() {
-		driver.get("http://localhost:5005");
+		driver.get("http://localhost:5005/include-exclude.html");
```

Comment:
Rather than changing this existing test, please add a new test all together. We should not update the existing test suite in order to add a new feature.

Reply:
Returned the original test and added test for several elements.